includes/abilities-api/class-wp-ability.php

```diff
@@ -234,12 +243,26 @@ protected function prepare_properties( array $args ): array {
 			);
 		}
 
+		// VALIDATE: Category is required and must be a string.
+		if ( empty( $args['category'] ) || ! is_string( $args['category'] ) ) {
```

Comment:
What's the reasoning for making categories required?

(Trying to put aside my general reservations about 1) categories as the correct shape and 2) implemented as a top-level arg instead of as `meta`, but that could be making it harder for me to intuit the reason here from the PR alone )